mobileapp/www/js/index.js

```diff
@@ -6,14 +6,25 @@
 
 document.addEventListener("deviceready", onDeviceReady, false);
 
+// Global variables on the fly
+// Intended for simple boolean values, NOT for passing details!
+// Usage: if(thing_to_set) {GLOBALS.foo = [true || false]}.
+var GLOBALS = {};
```

Comment:
So more something like inside `app` set `var has_p = true` then reference it as `app.has_p`?